internal/integration/unified/entity.go

```diff
@@ -503,6 +509,24 @@ func (em *EntityMap) close(ctx context.Context) []error {
 		}
 	}
 
+	// Clear automatically created collections used for queryable encryption
+	re := regexp.MustCompile("^enxcol_.*.e(sc|coc)$")
```

Comment:
(Optional) `regexp.MustCompile` panics if the regex can't be compiled. Consider moving this to a package-level initialization so any issue can be discovered at package initialization, not when the test is run.

E.g.
```go
// qeCollectionPattern matches collections automatically created for
// queryable encryption.
var qeCollectionPattern = regexp.MustCompile("^enxcol_.*.e(sc|coc)$")
```